api/sdmx-rest.yaml

```diff
@@ -618,10 +620,48 @@ components:
       schema:
         type: boolean
         default: false
+    offset:
+      in: query
+      name: offset
+      description: |
+        The number of observations (or series keys) to skip before beginning to return observations (or series keys).
+      required: false
+      schema:
+        type: integer
+        minimum: 0
+        default: 0
+    limit:
+      in: query
+      name: limit
+      description: |
+        The maximum number of observations (or series keys) to be returned. If no limit is set, all matching observations (or series keys) must be returned.
+      required: false
+      schema:
+        type: integer
+        minimum: 1
+    sort:
+      in: query
+      name: sort
+      description: |
+        This parameter specifies the order in which the returned data should be sorted. 
+
+        It contains either one or more component IDs, by which the data should be sorted, separated by `+` (to indicate an AND), the `*` operator, which represents all dimensions as positioned in the DSD, or the keyword `series_key`, which represents all non-time dimensions as positioned in the DSD. 
```

Comment:
Do we not foresee querying for data with another Dimension at obs? In such a case, the `series_key` would need to cover that, as well.

Reply:
Thanks, @stratosn. Addressed in https://github.com/sdmx-twg/sdmx-rest/pull/185/commits/ad7476a8e1e8f75fe46832b1ffe5e7ef8b1c0baa